extract-code.py

```diff
@@ -0,0 +1,30 @@
+import CommonMark
```

Comment:
What is this??? We already have tools for that.

Reply:
Please check gulpfile.js file.